src/theme/dojo/typeahead.m.css

```diff
@@ -5,3 +5,27 @@
 
 .triggerWrapper {
 }
+
+.arrow {
+	border: unset;
+	border-left: var(--border-width) solid var(--color-border);
+	bottom: var(--border-width);
+	color: var(--color-text-faded);
+	display: flex;
+	font-size: var(--font-size-icon);
+	line-height: var(--line-height-base);
+	padding: var(--spacing-regular);
+
+	cursor: pointer;
+	outline: none;
+
+	background: unset;
+
+	appearance: none;
+	-moz-appearance: none;
```

Comment:
do we need these? doesn't the build generate these for us? cc @matt-gadd

Reply:
It does not appear to. I just tested removing them and they are not in the end product.

Several other widgets have them in their css too.